readme.md

```diff
@@ -6,22 +6,23 @@
 
 <h4 align="center">
 
-  [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
-  [![Pull Requests Welcome](https://img.shields.io/badge/Pull%20Requests-welcome-brightgreen.svg)](https://github.com/janosh/awesome-svelte-kit/pulls)
-  [![License](https://img.shields.io/github/license/janosh/awesome-svelte-kit?label=License)](license)
-  [![Markdown linting](https://github.com/janosh/awesome-svelte-kit/workflows/Linter/badge.svg)](https://github.com/janosh/awesome-svelte-kit/actions)
+[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+[![Pull Requests Welcome](https://img.shields.io/badge/Pull%20Requests-welcome-brightgreen.svg)](https://github.com/janosh/awesome-svelte-kit/pulls)
+[![License](https://img.shields.io/github/license/janosh/awesome-svelte-kit?label=License)](license)
+[![Markdown linting](https://github.com/janosh/awesome-svelte-kit/workflows/Linter/badge.svg)](https://github.com/janosh/awesome-svelte-kit/actions)
 </h4>
 
 <br>
 
 1. [Official SvelteKit docs](https://kit.svelte.dev) [[code](https://github.com/sveltejs/sites/tree/master/sites/kit.svelte.dev)]
 2. [NYTimes Tracking the Coronavirus](https://nytimes.com/interactive/2021/us/new-york-city-new-york-covid-cases.html)
-     - uses: [Pancakes][]
+   - uses: [Pancakes][]
 3. [SK Incognito](https://sk-incognito.vercel.app) (the unofficial SvelteKit docs) [[code](https://github.com/GrygrFlzr/kit-docs)]
-     - uses: [MDsveX][], [Tailwind CSS][]
+   - uses: [MDsveX][], [Tailwind CSS][]
 4. [Geometric Bayes](https://svelte-geometric-bayes.netlify.app) [[code](https://github.com/janosh/svelte-geometric-bayes)]
-     - uses: [MDsveX][]
+   - uses: [MDsveX][]
 5. [TikZ](https://svelte-tikz.netlify.app) [[code](https://github.com/janosh/tikz)]
+6. [markushatvan.com](https://markushatvan.com) [[code](https://github.com/mhatvan/markushatvan.com)]
```

Comment:
How about we add this?

```md
   - uses: [Tailwind JIT][], [MDsveX][], blog posts with code snippets, contact form with [Svelte Forms Lib][], RSS feed and sitemap
```

and then

```md
[Tailwind JIT]: https://github.com/tailwindlabs/tailwindcss-jit
[Svelte Forms Lib]: https://github.com/tjinauyeung/svelte-forms-lib
```

at the bottom?

Reply:
Done, updated the PR with another commit

Reply:
Excellent!